lib/common/MongoDbLogger.ts

```diff
@@ -14,15 +18,25 @@ export default class MongoDbLogger {
     });
   }
 
-  static customLogger = function (_message: string | undefined, state: LoggerState | undefined): void {
-    switch (state?.type) {
+  /**
+   * The custom logger for general logging purpose in mongodb client
+   * @param _message The message to log
+   * @param state The complete logging event state
+   */
+  public static customLogger (_message: string | undefined, state: LoggerState | undefined): void {
+    if (state === undefined) {
+      return;
+    }
+
+    switch (state.type) {
       case 'warn':
         Logger.warn(state);
         break;
       case 'error':
         Logger.error(state);
         break;
-      case undefined:
+      case 'debug':
+        Logger.debug(state);
```

Comment:
Consider just reuse `.info()` because:
1. There we have not had the need to introduce yet another layer of logging, if something is generally useful, it's fine just let it be logged.
2. Our internal custom logger doesn't have concept of `.debug()` either, so more plumbing will be needed. Personally I don't think it's worth it.
```suggestion
        Logger.info(state);
```